src/maps/sources/OpenTopoSource.h

```diff
@@ -56,9 +58,13 @@ class OpenTopoSource: public img::TileSource {
 private:
     bool cancelToken = false;
     uint8_t hostIndex = 0;
-    std::string hosts = "abc";
     Downloader downloader;
-    std::string tileServer;
+    std::vector<std::string> tileServers;
+    std::string url;
+    size_t minZoom;
+    size_t maxZoom;
+    int tileWidth = 256;

```

Comment:
note that the tileWidth/tileHeight internally was already an int, and if I change it to size_t, I get a compilation warning in the `getTileDimensions` method:

https://github.com/cyberang3l/avitab/blob/configurable-online-slippy-maps-4/src/maps/sources/OpenTopoSource.cpp#L71C1-L73C2